modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-ocp-oauth.adoc
+:_module-type: PROCEDURE
+[id="configure-ocp-oauth-identity-provider_{context}"]
+= Configuring {ocp} Oauth server as an identity provider in {product-title}
+
+[role="_abstract"]
+Use the instructions in this section to integrate the built-in {ocp} OAuth server as an identity provider for {product-title}.
+
+.Prerequisites
+* You must have the `AuthProvider` permission to configure identity providers in {product-title}.
+* You must have already configured users and groups in {ocp} OAuth server.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Access Control*.
```

Comment:
This is the first instance of "RHACS" in this file - should it be in (RHACS) following one of the instances of {product-title} earlier?